### PR TITLE
Add Supply Cargo Attributes

### PR DESCRIPTION
**Pull request by ampersand38**
**When merged this pull request will:**
- Add attributes for supply vehicles to adjust their supply cargo

### CODE REVIEW

File: addons/attributes/stringtable.xml [hunk 1]
```diff
@@ -17,6 +17,18 @@
...
+            <English>Ammo Cargo</English>
+        </Key>
+        <Key ID="STR_ZEN_Attributes_FuelCargo">
+            <English>Fuel Cargo</English>
+        </Key>
+        <Key ID="STR_ZEN_Attributes_RepairCargo">
+            <English>Repair Cargo</English>
+        </Key>
+        <Key ID="STR_ZEN_Attributes_SupportCargo_Tooltip">
+            <English>Cargo to supply to other vehicles.</English>
```
Comment (mharis001) [inline addons/attributes/stringtable.xml:30]: Is this tooltip necessary?

Comment (ampersand38) [reply addons/attributes/stringtable.xml:30]: Can certainly be removed.

**Comment by mharis001** [comment]
Also, should add settings to enable/disable these like other attributes.

File: addons/attributes/stringtable.xml (hunk 1 above)
Comment (mharis001) [reply addons/attributes/stringtable.xml:30]: It sounds a bit too generic. I guess we can either remove or add more specific tooltips.

**Review by mharis001 — APPROVED** [review]
Code LGTM.

**Pull request merged**